tests/sentry/web/frontend/test_organization_auth_settings.py

```diff
@@ -606,7 +611,84 @@ def test_edit_sso_settings__scim(self):
         )
 
 
-dummy_provider_config = {
+@control_silo_test
+class OrganizationAuthSettingsSAML2Test(AuthProviderTestCase):
+    provider = DummySAML2Provider
+    provider_name = "saml2_dummy"
+
+    def setUp(self):
+        super().setUp()
+        self.user = self.create_user("foobar@sentry.io")
+        # self.organization = self.create_organization(owner=self.user, name="saml2-org")
+        # self.auth_provider_inst = AuthProvider.objects.create(
+        #     provider=self.provider_name,
+        #     config=dummy_provider_config,
+        #     organization_id=self.organization.id,
+        # )
+
+    def create_org_and_auth_provider(self, provider_name="saml2_dummy"):
+        self.user.update(is_managed=True)
+        with assume_test_silo_mode(SiloMode.REGION):
+            organization = self.create_organization(name="foo", owner=self.user)
+
+        auth_provider = AuthProvider.objects.create(
+            organization_id=organization.id, provider=provider_name
+        )
+        AuthIdentity.objects.create(user=self.user, ident="foo", auth_provider=auth_provider)
+        return organization, auth_provider
+
+    def create_om_and_link_sso(self, organization):
+        with assume_test_silo_mode(SiloMode.REGION):
+            om = OrganizationMember.objects.get(user_id=self.user.id, organization=organization)
+            setattr(om.flags, "sso:linked", True)
+            om.save()
+        return om
+
+    def test_edit_sso_settings(self):
```

Comment:
should we be checking that the x509 on the backend has changed? right now it seems like we're only looking at the audit log entry